src/ImageGallery.js

```diff
@@ -293,14 +293,33 @@ class ImageGallery extends React.Component {
     return 0;
   }
 
+  getThumbnailPositionClassName(thumbnailPosition) {
+    // get the specific thumbnailPosition className
+    const leftClassName = 'image-gallery-left';
+    const rightClassName = 'image-gallery-right';
+
+    switch (thumbnailPosition) {
+      case ('left'):
+        thumbnailPosition = ` ${leftClassName}`;
+        break;
+      case ('right'):
+        thumbnailPosition = ` ${rightClassName}`;
+        break;
+      default:
+        break;
+    }
+
+    return thumbnailPosition;
+  }
+
   getAlignmentClassName(index) {
     // Necessary for lazing loading
     const { currentIndex } = this.state;
     const { infinite, items } = this.props;
     let alignment = '';
-    const leftClassName = 'left';
-    const centerClassName = 'center';
-    const rightClassName = 'right';
+    const leftClassName = 'image-gallery-left';
+    const centerClassName = 'image-gallery-center';
+    const rightClassName = 'image-gallery-right';
```

Comment:
@xiaolin 
I used `image-gallery-<left|right|bottom|top>` instead of `image-gallery-thumbnails-<left|right|bottom|top>` because I found the original class names `<left|right|bottom|top>` are also used here (in `getAlignmentClassName` function). 
Should I:
1) also rename the class names here to `image-gallery-thumbnails-<left|right|bottom|top>`, or
2) keep the class names here as `image-gallery-<left|right|bottom|top>`, but style this class name the same as `image-gallery-thumbnails-<left|right|bottom|top>` ?

Thank you for your advice in advance!

Reply:
I think 2 is probably what we want. 

Update component to have 
```
image-gallery-<left|right|bottom|top>
image-gallery-thumbnails-<left|right|bottom|top>
``

Update scss file to differentiate the two new classes above.

e.g.

## old
```scss
.image-gallery-thumbnails-wrapper .left {}
.image-gallery-content .left {}
```

## new
```scss
.image-gallery-thumbnails-wrapper .image-gallery-thumbnails-left {}
.image-gallery-content .image-gallery-left {}
```